.github/ISSUE_TEMPLATE/core-team-story.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Story [Core team only - DONOT USE FOR COMMUNITY ISSUES]
+about: Create a User Story issue for eksctl Core Team
+title: '[Story] '
+labels: kind/story
+assignees: ''
+
+---
+
+### Description
+<!--
+Description of the Story issue, clearly explaining the problem we are solving here. 
+-->
+
+### Scope
+<!--
+List out what is in scope for this story
+-->
+
+### User story
+<!--
+As a User I want ....
+-->
+
+### Acceptance Criteria
+[] fizz
+[] buzz
+[] fizzbuzz
```

Comment:
```suggestion
- Given X 
- When Y 
- Then Z 
```
example:
Given a an EKS cluster
When I run `eksctl utils update-coredns`
Then the version of coredns running in the cluster should be updated to the latest

Reply:
updated 👍🏻